pandas/tests/io/formats/test_to_html.py

```diff
@@ -22,6 +22,15 @@
     pass
 
 
+@pytest.fixture
+def expected_html(read_file):
```

Comment:
Still somewhat confused as to why this needs to be a fixture versus just a plain old function - can't we just use it as the latter here?

Reply:
done.